packages.ini

```diff
@@ -301,9 +308,12 @@ python_versions = <3.11
 python_versions = <3.11
 [grpcio==1.48.1]
 python_versions = <3.11
+[grpcio==1.51.1]
+python_versions = <3.11
```

Comment:
can remove this line -- grpcio has wheels for 3.11 now

Reply:
@asottile-sentry this was added automatically by `add_pkg` - do we need an update there?

Reply:
`add_pkg` copies over the settings from previous versions as a convenience -- newer versions _sometimes_ have different requirements than previous versions

Reply:
👍🏻 was looking through the code and figured the same - should I just remove from this latest version or do the older versions have 3.11 wheels as well now?

Reply:
just the latest version :+1: